### ISSUE TITLE
Jitsi jitsi-2.10.5550-x64 hangs on splashscreen during startup

### ISSUE THREAD

**Issue by joerifeyen** [jitsi/jitsi#321]
Hi,

Today we upgraded our Jitsi client from jitsi-2.8.5426-x64 to jitsi-2.10.5550-x64 on Windows Server 2012R2.

When you start Jitsi it hangs on NimbuzzAvatars" on the splashscreen ".

![selection_059](https://cloud.githubusercontent.com/assets/12381725/22830372/8fa90eda-efa6-11e6-8947-462ba76882a3.png)

Deleting %appdata%\Jitsi doesn't help
Deleting %localappdata%\Jitsi doesn't help

Kr,

Joeri

**Comment by ibauersachs** [comment]
Can you please try to capture some console output with this command? It should create a jitsi.log textfile on your desktop.

```
cd "%ProgramFiles%\Jitsi"
jre\bin\java -classpath lib\felix.jar;sc-bundles\sc-launcher.jar;sc-bundles\dnsjava.jar;sc-bundles\util.jar;lib -Dfelix.config.properties=file:./lib/felix.client.run.properties -Djava.util.logging.config.file=lib/logging.properties -Djava.library.path=native -Dnet.java.sip.communicator.SC_HOME_DIR_NAME=Jitsi net.java.sip.communicator.launcher.SIPCommunicator > "%userprofile%\desktop\jitsi.log" 2>&1
```

**Comment by joerifeyen** [comment]
Please see attached file.
[jitsi.zip](https://github.com/jitsi/jitsi/files/776385/jitsi.zip)

**Comment by ibauersachs** [comment]
Unfortunately, I'm not able to reproduce this.

@joerifeyen Can you please check if the file "C:\Program Files\Jitsi\sc-bundles\jna.jar" is present and paste a list of its contents? If you don't have e.g. 7zip installed, you can copy it to the desktop and rename it to jna.zip to look inside.

@EUFC19 could you please also run the command from my second comment?

**Comment by joerifeyen** [comment]
The jna.jar is present in the "C:\Program Files\Jitsi\sc-bundles\jna.jar". The contents is: folder com and folder META-INF
[jna.zip](https://github.com/jitsi/jitsi/files/803124/jna.zip)

**Comment by lasselin** [comment]
Hi,

I got the same issue on ubuntu Xenial.
Cleaning ~/.jitsi doesn't solve it.
It hangs on "NimbuzzAvatars" on the splashscreen.

Best Regards,

**Comment by algorys** [comment]
Exactly the same issue with Jitsi 2.10.5550-x64 on Windows Server 2012. 

I got the same logs as @joerifeyen and get `jan.jar` also in my `%Program Files%\Jitsi\sc-bundles`.

I try to delete `%AppData%\Jitsi` and others. Reinstall after, no changes.

**Comment by ibauersachs** [comment]
Can you guys please check if pinning an application to the taskbar OR the latest nightly fixes this? The bug might have been caused by the issue fixed in #357.

**Comment by algorys** [comment]
Tested with `Jitsi-msi-2.11.5577` also on x86 and x64, but nothing change.

I've try to pin application to taskbar, delete again `%AppData%` directories, still the same errors.

**Comment by ibauersachs** [comment]
Can you please try [build 5580](https://download.jitsi.org/jitsi/nightly/windows/) (or newer) and lets us know?

**Comment by imho9** [comment]
still the same errors on build 5580 :(

**Comment by ibauersachs** [comment]
Is anyone here able to let me connect remotely to his machine?

**Comment by imho9** [comment]
wow, sorry! 
I replaced the library and jitsi was launched
Thank you!

**Comment by ibauersachs** [comment]
@imho9 What exactly did you replace?

**Comment by imho9** [comment]
Only felix.jar

**Comment by algorys** [comment]
I confim that `5580x64` is working for me too, on Windows Server 2012 R2.

Thanks @ibauersachs

**Comment by ibauersachs** [comment]
@imho9 Build 5580 has the updated felix.jar. Are you sure you tested with 5580 before you manually replaced a file? This would indicate a problem with updating the installed files during installs (which I doubt is an issue).

**Comment by imho9** [comment]
@ibauersachs it's my mistake. build 5580 worked too. Sorry :)

**Comment by Sylvain303** [comment]
Hi,

same problem on xubuntu 18.04.

Should I open a new issue?

```
$ dpkg -l | grep jitsi
ii  jitsi                                 2.10.5550-1                                 amd64        VoIP and Instant Messaging client
ii  jitsi-archive-keyring                 1.0.1                                       all          The public key for the Jitsi packages repository
```
attached log generated with

```
jitsi >> ~/tmp/jitsi.log 2>&1
```

[jitsi.log.gz](https://github.com/jitsi/jitsi/files/2471845/jitsi.log.gz)

ps in memory

```
sylvain  11055  5.1  0.9 9794184 150052 ?      Sl   07:45   1:56  |   \_ /usr/bin/java -classpath /usr/share/jitsi/lib/felix.jar:/usr/share/jitsi/sc-bundles/dnsjava.jar:/usr/share/jitsi/sc-bundles/sc-launcher.jar:/usr/share/jitsi/sc-bundles/util.jar:/usr/share/jitsi/lib -Djna.library.path=/usr/share/jitsi/lib/native -Dfelix.config.properties=file:/usr/share/jitsi/lib/felix.client.run.properties -Djava.util.logging.config.file=/usr/share/jitsi/lib/logging.properties -splash:splash.gif -Dnet.java.sip.communicator.SC_HOME_DIR_NAME=.jitsi net.java.sip.communicator.launcher.SIPCommunicator
```

log install via apt + jitsi repos https://jitsi.org/Main/DebianRepository

```
Start-Date: 2018-10-12  07:42:30
Commandline: /usr/bin/apt-get -y -o Dpkg::Options::=--force-confdef -o Dpkg::Options::=--force-confold install jitsi
Requested-By: sylvain (1000)
Install: ca-certificates-java:amd64 (20180516ubuntu1~18.04.1, automatic), default-jre-headless:amd64 (2:1.10-63ubuntu1~02, automatic), default-jre:amd64 (2:1.10-63ubuntu1~02, automatic), lib
atk-wrapper-java:amd64 (0.33.3-20ubuntu0.1, automatic), openjdk-11-jre-headless:amd64 (10.0.2+13-1ubuntu0.18.04.2, automatic), libatk-wrapper-java-jni:amd64 (0.33.3-20ubuntu0.1, automatic),
libgif7:amd64 (5.1.4-2, automatic), openjdk-11-jre:amd64 (10.0.2+13-1ubuntu0.18.04.2, automatic), fonts-dejavu-extra:amd64 (2.37-1, automatic), jitsi:amd64 (2.10.5550-1), jitsi-archive-keyri
ng:amd64 (1.0.1, automatic), java-common:amd64 (0.63ubuntu1~02, automatic)
End-Date: 2018-10-12  07:42:40
```

**Comment by damencho** [comment]
Try using java8

**Comment by ibauersachs** [comment]
The 2.10 stable release only works on Java 8. The nightly builds may or may not work on Java 9/10/11.

**Comment by kokbira** [comment]
The same for me on Linux Mint 19 (Tara)...

> Try using java8

But with above information I create a bash (jitsi.sh) with chmod +x on my desktop, and then I can run Jitsi without changing JDK of OS:

`#jitsi.sh:
echo -e "Alternate to JDK8 to run Jitsi, and then restore to JDK11";

#use "update-java-alternatives -l" to edit that

#change to JDK8...
sudo update-java-alternatives -s java-1.8.0-openjdk-amd64

#execute jitsi out of terminal, without waiting for it... 
nohup jitsi&

#restore to JDK11...
sudo update-java-alternatives -s java-1.11.0-openjdk-amd64

#remove output
rm nohup.out

#the end :)
exit
`

So dumb :p but I tried with PATH and JAVA_PATH with no success, so I used that dumb brute force, wawawa.

**Comment by kokbira** [comment]
> The 2.10 stable release only works on Java 8. The nightly builds may or may not work on Java 9/10/11.

At least I think a warning message could be launched to inform that to users:

> Hey, man, downgrade JDK and be happy.

**Comment by epoorklep** [comment]
And now instructions for newbie"s please .... 😁

**Comment by codehumper** [comment]
After much pain, I got jitsi to run on debian buster. I did the following steps:
1: Remove java 11 (or anything newer that jdk8 (did it in synaptic)  -this will also remove jicofo and jitsi
2: install OpenJDK8 per https://linuxize.com/post/install-java-on-debian-10/   --make sure to scroll down to the java8 install
3: I had jitsi_2.10.5550-1_amd64.deb  so used: # dpkg -i jitsi_2.10.5550-1_amd64.deb ...voila it worked!

hope this helps someone!
